lib/ttfunk/bin_utils.rb

```diff
@@ -0,0 +1,28 @@
+module TTFunk
+  module BinUtils
+    # assumes big-endian
+    def pack_int(arr, bit_width)
+      value = 0
+
+      arr.each_with_index do |element, index|
+        value |= element << bit_width * index
+      end
+
+      value
+    end
+
+    # assumes big-endian
+    def unpack_int(value, bit_width)
+      return [0] if value == 0
+      return [1] if value == 1
```

Comment:
Why is this a special case?

Reply:
The method needs to calculate how many bytes should be returned, so it takes the log base 2 of the value. `Math.log2(1) == 0.0`, which would mean 0 bytes in the return value.

Reply:
As I mentioned elsewhere, if this method took the size of unpacked value the calculation would be unnecessary.

Reply:
Acknowledged.